zomes/holo_acorn/code/src/lib.rs

```diff
@@ -514,6 +585,12 @@ mod holo_acorn {
 
     #[zome_fn("hc_public")]
     fn update_goal(goal: Goal, address: Address) -> ZomeApiResult<GetResponse<Goal>> {
+        let mut goal = goal;
+        let old_goal=hdk::utils::get_as_type::<Goal>( address.clone())?;
+        if goal.timestamp_updated==None
+        {goal.timestamp_updated=Some(goal.timestamp_created);}
```

Comment:
why set goal.timestamp_updated = Some(goal.timestamp_created) ?

Reply:
really is to not make so many changes in the ui